src/Microsoft.DocAsCode.HtmlToPdf/ConvertWrapper.cs

```diff
@@ -132,6 +132,18 @@ private void ConvertCore(string basePath)
                             AbsolutePathInTocPageFileTransformer(tocPageFilePath);
                             htmlModels.Insert(0, new HtmlModel { Title = _pdfOptions.TocTitle, HtmlFilePath = tocPageFilePath });
                         }
+
+                        var coverPage = FindCoverPageInManifest(manifest);
```

Comment:
Consider the following folder structure, where user want to generate 2 PDFs in one docfx run:
```
pdf
  |- cover.md
  |- toc.yml
pdf1
  |- cover.md
  |- toc.yml
```
Users should expect `pdf/toc.yml` to use `pdf/cover.md`, and `pdf1/toc.yml` to use `pdf1/cover.md`. So I think `FindCoverPageinManifest` should consider `tocFile` page, instead of the first one found globally. It will **only** pick up the `cover.md` alongside, like:
 ```cs
var coverPage = FindSiblingCoverPageInManifest(mainfest, tocFile)
```